docs/certbot.rst

```diff
@@ -0,0 +1,52 @@
+Certbot options
+```````````````
+
+The certbot playbook
+~~~~~~~~~~~~~~~~~~~~
+
+As a convenience, the Plone Ansible Playbook kit includes a separate
+playbook that will install certbot-nginx and create certificates as necessary for specified hostnames.
+
+The certbot playbook currently only supports Debian-family target servers.
+
+To use the certbot playbook, edit your ``local-configure.yml`` file to add a ``certbot_hosts`` list variable containing an entry for each hostname for which you wish to get a certbot certificate:
+
+.. code-block:: yaml
+
+   certbot_hosts:
+     - one.mcsmith.org
+     - two.mcsmith.org
+
+Run the playbook as you would the main playbook, adding whatever command-line switches you need (like ``-k`` or ``-K``):
+
+.. code-block:: console
+
+    ansible-playbook -k certbot.yml
+
+This will first install ``python3-certbot-nginx`` from the certbot/certbot ppa.
+Then it will create certificates as necessary for each hostname in the ``certbot_hosts`` list.
+If a certificate already exists, it will not attempt addition.
+
+Note that ``python3-certbot-nginx`` includes an auto-renewal cronjob.
+
+
+Webserver support
+~~~~~~~~~~~~~~~~~
+
+When the nginx role creates a configuration file for an virtual host, it will check ssl hostnames against the ``certbot_hosts`` list.
```

Comment:
```suggestion
When the nginx role creates a configuration file for a virtual host, it will check TLS hostnames against the ``certbot_hosts`` list.
```

If "ssl" is prefered, it should be fully capitalized.